File: 2-Semestre/Taller de Programación/Personal/Python-C#/Ejercicio 14.cs
/*
Usted cuenta con un seguro automotriz, el cual tiene un deducible de 5 UF.
Usted debe hacer uso del seguro por un accidente pequeño, el cual tiene un costo total de $678.987.
    - Mostrar por pantalla el monto que asume la seguradora y el monto que debe pagar usted.
    - Los montos se expresan en pesos (CLP).
*/

using System;

namespace Ejercicio_14
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Este programa calcula el monto que asume la aseguradora y el monto que debe pagar usted por un accidente automotriz\n");

            int costo_total = 678987;
            double uf;

            do
            {
                Console.Write("Ingrese el valor actual de la UF: ");
                uf = Convert.ToDouble(Console.ReadLine());

                if (uf <= 0)
                {
                    Console.WriteLine("[!] El valor de la UF no puede ser menor o igual a 0");
                }
            } while (uf <= 0);

            double aseguradora = uf * 5;
            double deducible = costo_total - aseguradora;

            Console.WriteLine($"El monto que asume la aseguradora es ${aseguradora}, usted deberá pagar ${deducible}");
        }
    }
}

File: 1-Semestre/Desarrollo de algoritmos/Actividades/Diagramas de flujo-Python (Semana 5)/Ejercicio 14.py
'''
Usted cuenta con un seguro automotriz, el cual tiene un deducible de 5 UF.
Usted debe hacer uso del seguro por un accidente pequeño, el cual tiene un costo total de $678.987.
    - Mostrar por pantalla el monto que asume la seguradora y el monto que debe pagar usted.
    - Los montos se expresan en pesos (CLP).
'''

print("Este programa calcula el monto que asume la aseguradora y el monto que debe pagar usted por un accidente automotriz.")
uf = float(input("Ingrese el valor actual de la UF: "))

while uf <= 0:
    print("[!] El valor de la UF debe ser positivo.")
    uf = float(input("Ingrese el valor actual de la UF: "))

aseguradora = uf * 5
deducible = 678987 - aseguradora
print(f"El monto que asume la aseguradora es ${aseguradora}, usted deberá pagar ${deducible}")